### DISCUSSION TITLE
holding shortcut key to seek, the top left blue bar isnt always updating

### DISCUSSION THREAD

**Discussion by liamauve** [iina/iina#4152] [General]
i have 3 pairs of keys bound to seek at different speed. when i hold a key, the top left blue bar will live update as long as im holding. but this only happen with 4 out of the 6 binds. the last 2 still seek, but both the time stamps above and the blue bar are static, it would only jump forward to the current position once i let go. 

the two keys that freeze are always the 2 listed in the playback menu. (step forward and step backward).
thanks for reading

**Comment by svobs** [comment]
This sounds like what I reported in issue #3729, but it's nice to get confirmation that others are experiencing it. It's not entirely clear what's causing the problem but it's likely to require a big code change. Hopefully I or someone else can get around to it soon.
